pkg/gofr/service/custom_header_test.go

```diff
@@ -0,0 +1,173 @@
+package service
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+
+	"gofr.dev/pkg/gofr/logging"
+)
+
+func Test_CustomDomainProvider_Get(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	path := "/path"
```

Comment:
unnecessary variable, since it is only being used once.
Please update in other places as well.